pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -902,3 +903,38 @@ func ValidateValidatingAdmissionPolicyUpdate(newC, oldC *admissionregistration.V
 func ValidateValidatingAdmissionPolicyBindingUpdate(newC, oldC *admissionregistration.ValidatingAdmissionPolicyBinding) field.ErrorList {
 	return validateValidatingAdmissionPolicyBinding(newC)
 }
+
+func validateValidatingAdmissionPolicyStatus(status *admissionregistration.ValidatingAdmissionPolicyStatus, fldPath *field.Path) field.ErrorList {
+	return validateTypeChecking(status.TypeChecking, fldPath.Child("typeChecking"))
```

Comment:
also need to validate conditions, when switching to metav1.Conditions, can call the metav1validation.ValidateConditions function